onboarding_documentation/technical_documentation/analyst_tutorial/analyst_tutorial.md

```diff
@@ -0,0 +1,122 @@
+As an analyst you will be running analysis jobs via `analysis-runner`, producing output, and then analysing the output. This tutorial will walk you through the process of running an analysis job, and then analysing the output.
+
+In this tutorial, we will begin with raw fastq files and process them using the large-cohort pipeline. This pipeline performs several steps:
+
+1. Aligns the reads to the reference genome.
+2. Conducts quality control checks.
+3. Calls variants and combines these variant calls.
+4. Performs an ancestry analysis.
+5. Plots the results of the ancestry analysis.
+
+While the large-cohort pipeline includes additional stages, for the purpose of this tutorial, we will only execute up to and including the Ancestry stage, which carries out Principal Component Analysis (PCA).
+
+For a description of what each stage of the `large-cohort` pipeline does, please see the `large-cohort` pipeline documentation [here](https://github.com/populationgenomics/production-pipelines#large-cohort-workflow)
+
+## Table of Contents
+- [1. Getting test data](#getting-test-data)
+- [2. Writing a config file](#writing-a-config-file)
+- [3. Submitting an analysis job](#submitting-an-analysis-job)
+- [4. Analysing the output](#analysing-the-output)
+
+## 1. Getting test data
+As an analyst it is vitally important we run our analysis jobs on test data before running them on real data. This is to ensure that the analysis job is working as expected, and that the output is as expected. This is also important to ensure that the analysis job is not going to take too long (and therefore cost too much!) to run on real data. As pipelines are constantly being updated, it is also important to run the analysis job on test data to ensure that the pipeline has not been broken by a recent update.
+
+For this tutorial we will be using the `bioheart` dataset, which is a collection of genomes from different sources. First things first, let's check if you have access to the `bioheart` dataset. To do this, run the following command:
```

Comment:
> a collection of genomes from different sources
BiohEART is a project that aims to identify people with atherosclerosis by finding genetic markers in blood before individuals acquire the disease, not a collection of genomes from different sources